test/regression/2875_connect_pub_to_sub_crash.cc

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2021 Amazon.com Inc or its affiliates. All rights reserved.
```

Comment:
Can we please change the copyright to use the same that in the rest of the code? Thanks.
```suggestion
 * Copyright (C) 2021 Open Source Robotics Foundation
```

Reply:
The way I understand copyright, this is how it is meant to be done. If somebody from OSRF also writes into the file, they can add a copyright line for OSRF.

Reply:
However, it's not a strong opinion - and it probably doesn't matter at all. I think you have write access to the branch if you really think it should be changed.

Reply:
I don't think that we have a policy on copyright holders anywhere in the Gazebo project so I was just suggestion to do what we have historically doing (a part from embedded pieces, all the files have the copyright assigned  to OSRF) but that does not give me any right to modify your submission. Please feel free to commit the suggestion or not, I will merge this tomorrow anyway. Thanks Emerson.